src/controllers/rpc/publish-rpc-controller.js

```diff
@@ -42,6 +42,7 @@ class PublishController extends BaseController {
             dataset: message.data.dataset,
             datasetRoot: message.data.datasetRoot,
             blockchain: message.data.blockchain,
+            isOperationV0: message.data.isOperationV0,
```

Comment:
why not make a new protocol version ?

Reply:
You could make v1_0_1HandleRequest, new commands and new protocol version ? Seems weird to use this flag

Reply:
yeah i agree it's weird and doesn't really fit the node architecture, tho i don't think the bloat added is too large since we won't even be maintaining code related to v0 (only used by ppl who didnt upgrade client to the new version)

we can do this or have a bunch of duplicated code since commands v100 and v101 are not too different, which is not ideal either imo

anyways good point, we'll discuss this more in depth and decide what to do, worst case we can leave it like this for the sake of time and rewrite it after v8 release